### ISSUE TITLE
Loss stucked in a local minimum

### ISSUE THREAD

**Issue by tomhayat** [AlexeyAB/darknet#1327]
Hi,

First thank you for your work, I have a loss around 2-3 just after 200 epochs and does not fluctuate after.

Do you know how is it possible ?

```
# Testing
batch=64
subdivisions=16
# Training
# batch=64
# subdivisions=16
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
flip=1
learning_rate=0.001
burn_in=200
max_batches = 500200
policy=steps
steps=500,1000
scales=.1,.1
```
This is my cfg file.
Thank you !

**Comment by AlexeyAB** [comment]
> First thank you for your work, I have a loss around 2-3 just after 200 epochs and does not fluctuate after.

* After 200 epochs or 200 iterations?
* What values can you get by using this command?
`./darknet detector calc_anchors data/obj.data -num_of_clusters 9 -width 416 -height 416`

* Try to use these params, and train about 11 000 iterations.
```
burn_in=1000
max_batches = 500200
policy=steps
steps=5000,10000
```

**Comment by tomhayat** [comment]
Thanks for your prompt response.
I cannot go lower than avg loss = 2.
I am training a one-class object detector. So I don't know if 11.
When I compute anchors I get :
```
 avg IoU = 71.95 %

Saving anchors to the file: anchors.txt
anchors = 35.4021,44.9469, 39.3446,71.4434, 69.8010,47.1530, 
41.4309,111.4425, 76.3810,82.7542, 115.9894,56.8419, 
176.7034,62.5068, 101.0442,154.7986, 285.5998,58.9830
```

I am trying your parameters and I will tell you how it went.

Thanks !

**Comment by PranjaLBiswas27** [comment]
Hi @tomhayat 

I see in the cfg file here there are 2 parameters. 1) Power, 2) Policy. What exactly are they? I have an idea that policy is how the learning rate changes, but I would like to know the exact polynomial that is used to control the learning rate over several epochs. Also, I dont know what exactly power is for. Please let me know. Thanks a lot in advance.

Kind Regards
Pranjal Biswas
